### ISSUE TITLE
"Failed to load Carpet Rule" log error spam on start

### ISSUE THREAD

**Issue by TheMrEngMan** [senseiwells/EssentialClient#111]
Minecraft version: 1.19.3
Fabric Loader version: 0.14.19
Carpet version: 1.4.96 (from [here](https://github.com/gnembon/fabric-carpet/releases/tag/1.4.96))
EssentialsClient version: 1.3.3 (from [here](https://modrinth.com/mod/essentialclient/version/gOCIekZc))

Tested on a brand new 1.19.3 instance with just the above mods.
The issue happens from the second launch of the game onward as EssentialsClient attempts to read it's config files.
I'm not sure how this can happen since some of the rules do not appear to be present in either code base.

Relevant part of log file:
[essentialsclienterrorspam.txt](https://github.com/senseiwells/EssentialClient/files/11194357/essentialsclienterrorspam.txt)
Generated CarpetClient.json upon first run (had to add .txt as GH doesn't seem to like .json files in issues)
[CarpetClient.json.txt](https://github.com/senseiwells/EssentialClient/files/11194531/CarpetClient.json.txt)

No other errors relating to EssentialsClient present in log, and the actual mod works fine in-game.

**Comment by Crec0** [comment]
You can ignore it. It will be fixed automatically.

**Comment by TheMrEngMan** [comment]
Thanks for the quick reply, upon further investigation, this seems to happen on every launch only on my main instance with other client mods present, and does indeed only happen once on the second launch of a fresh instance.
Looks like on of those other mods might be somehow interfering with the automatic fix process...

**Comment by Crec0** [comment]
It's caused by the parsing of carpet rules which is fetched from my database. You don't need to worry about or do any digging, we are aware of it and I am one responsible for fixing it. I am just procrastinating :p

**Comment by TheMrEngMan** [comment]
Thanks for clarifying! I was beginning to wonder why no matter what I did the errors persisted.
Maybe it was a fluke that on the third launch of my new test instance the errors were not present, therefore I thought it was something to do with my main instance. After more testing I can confirm it happens on every launch.

I will leave it up to you to close the issue when it's solved on your end then :)
In the meantime I've mitigated this in my local build (omitted the exception from the LOGGER.error call).
